bench_method_test.go

```diff
@@ -301,10 +303,92 @@ func TestBenchmarkMethodWarmTransportsError(t *testing.T) {
 	}
 }
 
-func TestBenchmarkMethodHealth(t *testing.T) {
-	m := benchmarkMethodForROpts(t, RequestOptions{
-		Encoding: encoding.Thrift,
-		Health:   true,
-	}, transport.TChannel)
-	assert.Equal(t, "Meta::health", m.Method())
```

Comment:
Why did you remove this test?

Reply:
Benchmark method no longer exposes `Method`, so this test wouldn't be valid anymore